app/importers/import_task.rb

```diff
@@ -156,6 +156,8 @@ def run_update_tasks
   end
 
   def log(msg)
-    @log.write("\n\n💾  ImportTask: #{msg}")
+    full_msg = "\n\n💾  ImportTask: #{msg}"
+    @log.write full_msg
+    @record.log += full_msg
```

Comment:
This line seems like the most meaningful change.  We'll keep these logs indefinitely, in contrast to the rolling expiration with other logs.  And adding the log output to the record increases the record size and grows the `ImportRecords` table at that rate each day.  These sound fine, but both are slightly different than how I usually think about logs.

This also introduces a new choice to developers - if someone wants to see the logs for what happened in an import task, should they look here in the UI or in the logs in logentries?  Both "products" for looking at logs have different features and affordances (eg, sorting, grepping, seeing historical comparisons, etc).  It's good how you flagged this as maybe 🚴 shedding and when it starts to feel like you're re-creating another product it's good to ask if it's worthwhile (and sometime it definitely is!).  Another intermediate option might be to have this UI link directly to the relevant bits in logentries (maybe possible, not sure).

Anyway, to avoid more 🚴s, I'd say from here you should comment and do whatever you think is best and  🚢it!  👍

Reply:
Yeah, agreed this is the most meaningful change! 

I want to create a "one-stop shop" for debugging and monitoring the nightly import jobs. Right now it's a mix of queries in the console, Logentries, maybe error alert emails from the job, and the Import Record page in the app. By adding a copy of the logs to the Import Record page my goal is to make one page that can answer the question, "what's going on with the import job?" 

When I mentioned 🚲shedding, I was just referring to the HTML => React part of the work in particular.

Reply:
👍